srcpkgs/python3-et-xmlfile/update

```diff
@@ -0,0 +1,2 @@
+site=https://foss.heptapod.net/openpyxl/et_xmlfile/-/tags
+pattern='/archive/[^/]+/\Q'et_xmlfile'\E-v?\K[\d.]+(?=\.tar\.gz")'
```

Comment:
settings `pkgname=${pkgname%python3-}` (instead of overriding `pattern`) is probably enough for both of these update files

Reply:
I guess you mean `pkgname=${pkgname#python3-}`
applied, thanks.
for `python3-et-xmlfile`, hardcoded instead because also need `_` instead of `-`.